src/llmq/utils.cpp

```diff
@@ -1019,7 +1009,7 @@ bool IsQuorumTypeEnabledInternal(Consensus::LLMQType llmqType, const CQuorumMana
             return VersionBitsState(pindex, consensusParams, Consensus::DEPLOYMENT_TESTDUMMY, llmq_versionbitscache) == ThresholdState::ACTIVE;
         }
         case Consensus::LLMQType::LLMQ_100_67:
-            return pindex != nullptr && pindex->nHeight + 1 >= consensusParams.DIP0020Height;
+            return pindex->nHeight + 1 >= consensusParams.DIP0020Height;
```

Comment:
same, a change in behaviour here but seems to be working.

Reply:
Other switch cases fail if nullptr so seems acceptable.